pkg/systemd/overview-cards/lastLogin.jsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2021 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+import React from 'react';
+import { Flex, FlexItem } from '@patternfly/react-core';
+import { InfoCircleIcon } from "@patternfly/react-icons";
+
+import cockpit from "cockpit";
+import { LoginMessage } from "../loginMessage.jsx";
+
+import './lastLogin.scss';
+
+const _ = cockpit.gettext;
+
+export class LastLogin extends React.PureComponent {
+    render() {
+        const { loginMessages } = this.props;
+
+        if (loginMessages && !('last-login-host' in loginMessages)) {
+            return null;
+        }
+
+        return (
+            <li className="last-login" id="page_status_last_login">
+                <Flex flexWrap={{ default: 'nowrap' }}>
+                    <FlexItem spacer={{ default: 'spacerSm' }}><InfoCircleIcon size="sm" /></FlexItem>
+                    <FlexItem spacer={{ default: 'spacerMd' }} className="system-information-last-login pf-u-text-nowrap">{_("Last login")}</FlexItem>
```

Comment:
This classes need the corresponding CSS files to be imported.

``` diff
diff --git pkg/systemd/overview-cards/lastLogin.scss pkg/systemd/overview-cards/lastLogin.scss
index 43736ac20..a027d1673 100644
--- pkg/systemd/overview-cards/lastLogin.scss
+++ pkg/systemd/overview-cards/lastLogin.scss
@@ -1,3 +1,6 @@
+@import "_global-variables.scss";
+@import '@patternfly/patternfly/utilities/Text/text.scss';
+
 .system-information-last-login {
     font-weight: var(--pf-global--FontWeight--semi-bold);
 }
```

Add this to lastLogin.scss